core/sis-referencing/src/test/java/org/apache/sis/geometry/CoordinateFormatTest.java

```diff
@@ -188,4 +193,76 @@ public void testParseGeographic4D() throws ParseException {
                          e.getLocalizedMessage());  // In the language specified at CoordinateFormat construction time.
         }
     }
+
+    @Test
+    public void testCreatesCoordinateFormatTaking2ArgumentsAndCallsClone() {
+        Locale locale = Locale.CANADA;
+        CoordinateFormat coordinateFormat = new CoordinateFormat(locale, null);
+        CoordinateFormat coordinateFormatTwo = coordinateFormat.clone();
+
+        assertNotSame(coordinateFormatTwo, coordinateFormat);
+    }
+
+    @Test
+    public void testParseAndFormatTaking2Arguments() throws IOException, ParseException {
+        CoordinateFormat coordinateFormat = new CoordinateFormat();
+        DirectPosition2D directPosition2D = new DirectPosition2D();
+        StringBuffer stringBuffer = new StringBuffer();
+        coordinateFormat.format(directPosition2D, stringBuffer);
+        ParsePosition parsePosition = new ParsePosition(0);
+        DirectPosition directPosition = coordinateFormat.parse(stringBuffer, parsePosition);
+
+        assertEquals(2, directPosition.getDimension());
+        assertEquals(2, directPosition.getCoordinate().length);
+        assertEquals(0.0, directPosition.getCoordinate()[0], 0.01);
+        assertEquals(0.0, directPosition.getCoordinate()[1], 0.01);
+    }
+
+    @Test
+    public void testParseAndSetDefaultCRSAndParse() throws ParseException {
+        CoordinateFormat coordinateFormat = new CoordinateFormat();
+        VerticalCRSMock verticalCRSMock = (VerticalCRSMock) VerticalCRSMock.BAROMETRIC_HEIGHT;
+        coordinateFormat.setDefaultCRS(verticalCRSMock);
+        ParsePosition parsePosition = new ParsePosition(4);
+        DirectPosition directPosition = coordinateFormat.parse("UTF-8", parsePosition);
```

Comment:
"UTF-8" string here is misleading. This is an encoding name, which has nothing to do with coordinates. I understand that the intend was to parse the "8" after "UTF-8", but it is better to use a string that reflect better the intend, for example "[skipped] 8" - this make clear that we test the capability to skip the "[skipped]" part.